app/README.md

```diff
@@ -1,25 +1,51 @@
-# CDOGS Backend
-CDOGS (Common Document Generation Service) Backend is a simple node.js wrapper around the [BC Gov. Common Document Generation Service](https://github.com/bcgov/common-document-generation-service.git).  The wrapper is a simple pass through to the service, its main purpose is to hide the service client credentials from the frontend.  
+# DGRSC - Document Generation Service Showcase Backend
+DGRSC (Document Generation Service Showcase) Backend is a simple node.js wrapper around the [BC Gov. Common Document Generation Service](https://github.com/bcgov/common-document-generation-service.git).  The wrapper is a simple pass through to the CDOGS service, its main purpose is to hide the service client credentials from the frontend.  We host the showcase frontend from this node.js backend.  
```

Comment:
Even though the .git part of the url works correctly, it might be better to drop that part as we're expecting a hyperlink to the page, not the raw git repository (though Github handles that seamlessly).